matplotcheck/tests/test_timeseries_module.py

```diff
@@ -1,9 +1,152 @@
 import pytest
+from matplotcheck.base import PlotTester
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from io import StringIO
+
+
+"""Fixtures"""
+
+
+@pytest.fixture
+def pd_df_timeseries():
+    """Create a pandas dataframe for testing, with timeseries in one column"""
+    return pd.DataFrame(
+        {
+            "time": pd.date_range(start="1/1/2018", periods=100),
+            "A": np.random.randint(0, 100, size=100),
+        }
+    )
+
+
+@pytest.fixture
+def pd_df_timeseries_low():
+    """Create a pandas dataframe for testing, with timeseries in one column,
+    with dates before 1970."""
+    return pd.DataFrame(
+        {
+            "time": pd.date_range(start="1/1/1900", periods=100),
+            "A": np.random.randint(0, 100, size=100),
+        }
+    )
+
+
+@pytest.fixture
+def pd_df_timeseries_csv():
```

Comment:
For whatever reason, the datetime data in the homework file was being converted by matplotlib completely differently (i.e. using a different epoch). I have not idea what causes this difference. (It is not because the dates are before 1970. - in other cases, it will still use the 1970 epoch.)

This fixture imports similar data as if it was importing from a CSV, and for whatever reason, it is a reproducible way for matplotlib to use the other epoch.